lib/elix/responders/guys.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Elix.Responders.Guys do
+  @moduledoc """
+  Encourages the use of gender-neutral forms of address
+  """
+
+  use Hedwig.Responder
+
+  @forms_of_address [
+    "all y'all",
+    "crew",
+    "exlisirists",
+    "everybody",
+    "everyone",
+    "folks",
+    "friends",
+    "mere mortals",
+    "non-robots",
+    "peeps",
+    "y'all"
+  ]
+
+  @usage """
+  (hey|you|those) guys - Replies with a gentle reminder to use gender-neutral forms of address
+  """
+
+  # hear ~r/((hey|hi|those|yo|you) guys|\Aguys)(\W|\z)/i, msg do
```

Comment:
I like the solution you went with, so I think it'd be good to remove this commented-out alternative